tests/Functional/Command/ProxySyncReleasesCommandTest.php

```diff
@@ -109,7 +108,7 @@ private function fakeLockFactory(): LockFactory
 
     private function lockFactory(): LockFactory
     {
-        /** @var Connection */
+        /** @var \PDO */
```

Comment:
why not connection?

Reply:
Good catch, this is was something I had to look into. The `PdoStore` is now `PDO` only, had to change this to `DoctrineDbalStore` instead and updated the `@var` doc comment back to `\Doctrine\DBAL\Connection`